templates/table-maps.html

```diff
@@ -1,139 +1,167 @@
-			<td>{{.UserStatsForGame.Stats.TotalWinsMapCsOffice}}</td>
-			<td></td>
-		</tr>
-		<tr>
-			<td>TotalRoundsMapDeAztec</td>
-			<td>{{.UserStatsForGame.Stats.TotalRoundsMapDeAztec}}</td>
-			<td>{{.UserStatsForGame.Stats.TotalWinsMapDeAztec}}</td>
-			<td></td>
-		</tr>
-		<tr>
-			<td>TotalRoundsMapDeBank</td>
-			<td>{{.UserStatsForGame.Stats.TotalRoundsMapDeBank}}</td>
-			<td>{{.UserStatsForGame.Stats.TotalWinsMapDeBank}}</td>
-			<td></td>
-		</tr>
-		<tr>
-			<td>TotalRoundsMapDeCbble</td>
-			<td>{{.UserStatsForGame.Stats.TotalRoundsMapDeCbble}}</td>
-			<td>{{.UserStatsForGame.Stats.TotalWinsMapDeCbble}}</td>
-			<td></td>
-		</tr>
-		<tr>
-			<td>TotalRoundsMapDeDust</td>
-			<td>{{.UserStatsForGame.Stats.TotalRoundsMapDeDust}}</td>
-			<td>{{.UserStatsForGame.Stats.TotalWinsMapDeDust}}</td>
-			<td></td>
-		</tr>
-		<tr>
-			<td>TotalRoundsMapDeDust2</td>
-			<td>{{.UserStatsForGame.Stats.TotalRoundsMapDeDust2}}</td>
-			<td>{{.UserStatsForGame.Stats.TotalWinsMapDeDust2}}</td>
-			<td></td>
-		</tr>
-		<tr>
-			<td>TotalRoundsMapDeInferno</td>
-			<td>{{.UserStatsForGame.Stats.TotalRoundsMapDeInferno}}</td>
-			<td>{{.UserStatsForGame.Stats.TotalWinsMapDeInferno}}</td>
-			<td></td>
-		</tr>
-		<tr>
-			<td>TotalRoundsMapDeLake</td>
-			<td>{{.UserStatsForGame.Stats.TotalRoundsMapDeLake}}</td>
-			<td>{{.UserStatsForGame.Stats.TotalWinsMapDeLake}}</td>
-			<td></td>
-		</tr>
-		<tr>
-			<td>TotalRoundsMapDeNuke</td>
-			<td>{{.UserStatsForGame.Stats.TotalRoundsMapDeNuke}}</td>
-			<td>{{.UserStatsForGame.Stats.TotalWinsMapDeNuke}}</td>
-			<td></td>
-		</tr>
-		<tr>
-			<td>TotalRoundsMapDeSafehouse</td>
-			<td>{{.UserStatsForGame.Stats.TotalRoundsMapDeSafehouse}}</td>
-			<td>{{.UserStatsForGame.Stats.TotalWinsMapDeSafehouse}}</td>
-			<td></td>
-		</tr>
-		<tr>
-			<td>TotalRoundsMapDeShorttrain</td>
-			<td>{{.UserStatsForGame.Stats.TotalRoundsMapDeShorttrain}}</td>
-			<td>{{.UserStatsForGame.Stats.TotalWinsMapDeShorttrain}}</td>
-			<td></td>
-		</tr>
-		<tr>
-			<td>TotalRoundsMapDeStmarc</td>
-			<td>{{.UserStatsForGame.Stats.TotalRoundsMapDeStmarc}}</td>
-			<td>{{.UserStatsForGame.Stats.TotalWinsMapDeStmarc}}</td>
-			<td></td>
-		</tr>
-		<tr>
-			<td>TotalRoundsMapDeSugarcane</td>
-			<td>{{.UserStatsForGame.Stats.TotalRoundsMapDeSugarcane}}</td>
-			<td>{{.UserStatsForGame.Stats.TotalWinsMapDeSugarcane}}</td>
-			<td></td>
-		</tr>
-		<tr>
-			<td>TotalRoundsMapDeTrain</td>
-			<td>{{.UserStatsForGame.Stats.TotalRoundsMapDeTrain}}</td>
-			<td>{{.UserStatsForGame.Stats.TotalWinsMapDeTrain}}</td>
-			<td></td>
-		</tr>
-		<tr>
-			<td>TotalRoundsMapDeVertigo</td>
-			<td>{{.UserStatsForGame.Stats.TotalRoundsMapDeVertigo}}</td>
-			<td>{{.UserStatsForGame.Stats.TotalWinsMapDeVertigo}}</td>
-			<td></td>
-		</tr>
-	</tbody>
-</table>
+<div class="row">
+	<div class="col s12">
+		<table id="mapTable">
+			<thead>
+				<tr>
+					<th>Played Map</th>
+					<th>Rounds</th>
+					<th>Rounds Won</th>
+					<th>Win Rate</th>
```

Comment:
Would it be possible to color the win rate as csgo-stats.com does?
- Under 50%: Red
- 50% and over: Green

I think this would make a nice addition and provide clarity for player on their "problematic maps" they need to work on. Maybe take  a look if it can be done without too much effort